packages/entrykit/src/onboarding/quarry/Allowance.tsx

```diff
@@ -30,7 +29,7 @@ export function Allowance({ isActive, isExpanded, userAddress }: Props) {
         claimGasPass.status === "idle" &&
         allowance.isSuccess &&
         allowance.data != null &&
-        allowance.data < minGasBalance
+        allowance.data < parseEther("0.01")
```

Comment:
this is only used to trigger an automatic claim of a gas pass in the quarry paymaster when balance dips below some amount